src/pages/AddProductPage/components/FileInput/FileInput.jsx

```diff
@@ -0,0 +1,59 @@
+import React, { useEffect, useRef, useState } from "react";
+import "./FileInput.css";
+import { FaPlus } from "react-icons/fa6";
+import { IoClose } from "react-icons/io5";
+
+function FileInput({ name, value, onChange }) {
+  const [preview, setPreview] = useState();
+  const inputRef = useRef();
+  const handleFileChange = (e) => {
+    const nextValue = e.target.files[0];
+    onChange(name, nextValue);
+  };
+  const handleDeletePreviewImg = () => {
+    onChange(name, null);
+  };
+  useEffect(() => {
+    if (!value) return;
+
+    const nextPreview = URL.createObjectURL(value);
+    setPreview(nextPreview);
+
+    return () => {
+      setPreview();
```

Comment:
```suggestion
      setPreview(undefined or null);
```

명시적으로 undefined 또는 null을 넣어주는게 가독성이 더 좋은 것 같습니다.
아무것도 없으면 실수로 작성한 코드같아요 ㅎㅎ